### DISCUSSION TITLE
Example of online ide(StackBlitz) not running

### DISCUSSION THREAD

**Discussion by baboon-king** [remix-run/react-router#9594] [General]
mian: https://github.com/remix-run/react-router/tree/dev/examples/notes#preview

I got a error Page
![image](https://user-images.githubusercontent.com/63645337/201812595-3ce53c8c-3ea4-4720-ad2f-bccfb5d1d9c8.png)

**Comment by mcansh** [comment]
fixed by #9596, thanks for the heads up 😄
